node/cmd/guardiand/node.go

```diff
@@ -1456,6 +1511,42 @@ func runNode(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	// A check above does not allow this in mainnet.
+	if shouldStart(arbitrumSepoliaRPC) {
+		wc := &evm.WatcherConfig{
+			NetworkID: "arbitrumsepolia",
```

Comment:
should these match the chain names we give in sdk?

Reply:
For consistency, I'd prefer it. Whichever gets the new chain first is the "canonical" source of truth.